src/main/java/org/bihealth/mi/easybus/implementations/http/AuthHandler.java

```diff
@@ -0,0 +1,24 @@
+package org.bihealth.mi.easybus.implementations.http;
```

Comment:
Here and other classes: License header is missing. Please check all (new) classes.